tests/torch/nas/test_elastic_width.py

```diff
@@ -147,6 +154,40 @@ def test_width_reorg(basic_model):
     compare_tensors_ignoring_the_order(after_reorg, before_reorg)
 
 
+def test_width_custom_external_reorg(basic_model):
+    if not isinstance(basic_model, TwoSequentialFcLNTestModel):
+        pytest.skip("Skip test for conv model")
```

Comment:
What are the limitations for conv models? Are you going to resolve it?

Reply:
There aren't limitations. We have added more test. Thanks!